README.md

```diff
@@ -34,6 +34,7 @@ The [`my-app/.templates`](.templates) dir contains some templates you can use in
 
 - `<APP_NAME>`: the application name
 - `<APP_SHORT_DESCRIPTION>`: a quick app description
+- `<APP_CATEGORY>`: the app category (used in the cozy-bar apps navigation)
```

Comment:
You convinced me, I add more details in https://github.com/cozy/cozy-template/pull/43/commits/409afb6c274fd7dedc109c1e5c4816fb9dcfd0bd 👍